vmdb/spec/requests/api/tag_collections_spec.rb

```diff
@@ -7,21 +7,35 @@
 
   include Rack::Test::Methods
 
-  before(:each) do
-    init_api_spec_env
+  let(:zone)         { FactoryGirl.create(:zone, :name => "api_zone") }
+  let(:miq_server)   { FactoryGirl.create(:miq_server, :guid => miq_server_guid, :zone => zone) }
+  let(:ems)          { FactoryGirl.create(:ems_vmware, :zone => zone) }
+  let(:host)         { FactoryGirl.create(:host) }
+
+  let(:tag1)         { {:category => "department", :name => "finance", :path => "/managed/department/finance"} }
```

Comment:
yes and no, there are more tags created, and I only want these particular ones. Would still need to search by name (really only path in Tag) and parse out category and name.  not worth the extra code.